.gitignore

```diff
@@ -6,6 +6,11 @@ docs/research/data/**
 public/
 credentials*
 
+*pem
```

Comment:
**suggestion (non-blocking)**: If these are meant to match extensions, maybe include the dot in the pattern for microscopic improvement in false matches?

Reply:
Good suggestion, done.